.github/workflows/test-drupal.yml

```
@@ -51,6 +51,19 @@ jobs:
         working-directory: tools/sdc
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Validate SDC components
+        run: |
+          ./build/vendor/bin/drush ct_dev:validate-components
+        run: |
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:56,col:9

(syntax-check)